dev/devicelab/lib/tasks/microbenchmarks.dart

```diff
@@ -122,11 +123,27 @@ Future<Map<String, double>> _readJsonResults(Process process) {
     }
 
     if (line.contains(jsonEnd)) {
+      final String jsonOutput = jsonBuf.toString();
+
+      // If we end up here and have already parsed the results, it suggests that
+      // we have recieved output from another test because our `flutter run`
+      // process did not terminate correctly.
+      // https://github.com/flutter/flutter/issues/19096#issuecomment-402756549
+      if (resultsHaveBeenParsed) {
+        throw 'Additional JSON was received after results has already been '
+            'processed. This suggests the `flutter run` process may have lived '
```

Comment:
minor nit: please try to keep expressions vertically aligned (so in this case, making the leading apostrophes all be in the same vertical column); it makes reading the code much easier (because you can tell at a glance that it's one expression and how they nest).

Reply:
Makes sense - thanks! I updated here -> https://github.com/flutter/flutter/pull/19256